cmd/minikube/cmd/start.go

```diff
@@ -390,6 +390,11 @@ func validateKubernetesVersions(old *cfg.Config) string {
 	if nv == "" {
 		nv = constants.DefaultKubernetesVersion
 	}
+
+	if nv[0] != version.VersionPrefix {
+		exit.WithCode(exit.Data, "the --kubernetes-version flag requires the version to start with a 'v'")
```

Comment:
small nitpick: Do you mind making this exit.Usage instead of exit.Data? Since the input data is from a command-line, it seems to be more correct:

```
EX_USAGE (64)	   The command was used	incorrectly, e.g., with	the
			   wrong number	of arguments, a	bad flag, a bad	syntax
			   in a	parameter, or whatever.
```

Thanks!

Reply:
done. thanks!